apps/api/src/domain/models/song/commands/song-created.event.ts

```diff
@@ -0,0 +1,25 @@
+import { AggregateId } from '../../../types/AggregateId';
+import BaseDomainModel from '../../BaseDomainModel';
+import { AddSong } from './add-song.command';
+
+/**
+ * For now, we are just mirroring our command payload onto our events. We can update
+ * this if and when we decide to use the events (e.g. for event sourcing).
+ */
+export class SongCreated extends BaseDomainModel {
+    type = 'SONG_CREATED';
+
+    dateCreated: number;
+
+    id: string;
+
+    constructor(command: AddSong, eventId: AggregateId) {
+        super();
+
+        Object.assign(this, command);
```

Comment:
Just curious why you have this instead of a `command` property.  I'm guessing this has to do with how the command itself is applied?